test/testground/compositions/standard/plan.toml

```
@@ -0,0 +1,54 @@
+[metadata]
+name = "standard"
+author = "core-app"
+
+[global]
+plan = "core-app"
```

Following this comment, may want to consider renaming it to 
```suggestion
plan = "celestia"
```